images/win/scripts/Installers/Validate-Toolset.ps1

```diff
@@ -78,6 +79,11 @@ foreach($tool in $tools) {
     $toolExecs = $toolsExecutables[$tool.name]
 
     foreach ($version in $tool.versions) {
+        # Add wildcard if missing
+        if (-not $version.Contains('*')) {
+            $version += '.*'
+        }
+
         # Check if version folder exists

```

Comment:
Is it possible to improve PyPy documentation to put not only version of Python but version of PyPy too?
`7.3.1` for now.
It is request from customer: https://github.com/actions/virtual-environments/issues/638

Reply:
This version can be parsed from `pypy3 --version`